rules/manager_test.go

```diff
@@ -326,6 +326,21 @@ func TestForStateAddSamples(t *testing.T) {
 	}
 }
 
+// sortAlerts sorts `[]*Alert` w.r.t. the Labels.
+// This is cleaner for small slice than adding Len(), Less(), Swap() for []*Alert.
+func sortAlerts(items []*Alert) {
+	var n = len(items)
+	for i := 1; i < n; i++ {
+		j := i
+		for j > 0 {
+			if items[j-1].Labels.Hash() > items[j].Labels.Hash() {
+				items[j-1], items[j] = items[j], items[j-1]
+			}
+			j = j - 1
+		}
+	}
+}
+
 func TestForStateRestore(t *testing.T) {
 
```

Comment:
Not part of this PR, but why the extra whitespace?

Reply:
Where exactly?

Reply:
Right after the function start :)